### ISSUE TITLE
Unable to get EpiFusion 0.0.3 to run (MacOS)

### ISSUE THREAD

**Issue by tgvaughan** [ciarajudge/EpiFusion#12]
Hi Ciara, I was just trying the latest version out and couldn't seem to get anywhere.

Firstly, running without arguments directly produces an error.  Maybe instead print usage information in this case?
```
$ java -jar EpiFusionv0.0.3.jar 
EpiFusion
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at Main.main(Main.java:23)
```

Secondly, running with example XML from repository also produces an error.  I'm sure I'm just doing something very silly, but in case not here is the stack trace:
```
java -jar EpiFusionv0.0.3.jar introduction-example.xml 
EpiFusion
Exception in thread "main" java.lang.NumberFormatException: For input string: "
"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:651)
	at java.base/java.lang.Integer.parseInt(Integer.java:781)
	at Incidence.<init>(Incidence.java:21)
	at Main.parseXMLInput(Main.java:98)
	at Main.main(Main.java:23)
```

Thanks!
Tim

**Comment by ciarajudge** [comment]
Thanks Tim! Yeah you've managed to catch me right in the middle of the transition from v0.0.3 to v1.0 so the wiki and current executable file are a bit out of sync - hopefully by mid/end of next week this should be all resolved. I also am in the midst of adding error handling for common errors (such as running with no arguments). I'll comment back here when i've resolved these bits

**Comment by tgvaughan** [comment]
Ok, thanks for the clarification!

**Comment by ciarajudge** [comment]
Hey Tim! There's a new release available now (v0.0.4) that should mesh nicely with 'Tutorial 1: Running an existing XML file'. This release also has some better error handling, but it's far from comprehensive at this point; let me know if you come across any more issues. Currently I'm putting together a more comprehensive real data tutorial too.
